test/Docker.DotNet.Tests/ISystemOperations.Tests.cs

```diff
@@ -17,35 +18,54 @@ public ISystemOperationsTests()
             _client = new DockerClientConfiguration().CreateClient();
         }
 
-        [SupportedOSPlatformsFact(Platform.Windows)]
-        public void DockerService_IsRunning()
+        [Fact]
+        public void Docker_IsRunning()
         {
-            var services = ServiceController.GetServices();
-            using (var dockerService = services.SingleOrDefault(service => service.ServiceName == "docker"))
-            {
-                Assert.NotNull(dockerService); // docker is not running
-                Assert.Equal(ServiceControllerStatus.Running, dockerService.Status);
-            }
+            var dockerProcess = Process.GetProcesses().FirstOrDefault(_ => _.ProcessName == "docker");

```

Comment:
Now I'm not sure it will work on Windows as well 😄 

It should check for `docker` and `docker.exe` in order to support all OSes. Or just use `Contains()` and `IgnoreCase`.

Reply:
Yeah, `Contains()` would be better.

Reply:
For sure `Contains()`. I just confirmed that on _real_ docker services the process is called `dockerd` as @jterry75  mentioned. 

However, on desktops it is called `docker` or `docker.exe` depending on the OS. So `Contains()` would work just fine for all the cases.